tools/spec-gen-sdk/src/automation/reportStatus.ts

```diff
@@ -174,24 +182,24 @@ export const renderFilterLog2HTML = async (context: WorkflowContext) => {
         const renderNoteBlock = blockString && htmlBlockTemp(blockString);
         totalBody += (renderNoteBlock + bodyString);
     }
-    
 
-    const htmlString: string = htmlTemp(totalBody);
+
+    const htmlString: string = htmlTemp(totalBody, `spec-gen-sdk ${context.config.sdkName}`);
```

Comment:
context.config.sdkName would be azure-sdk-for-{language}, need to split it to get the language. The title should be something like 'spec-gen-sdk-net result'.